cpp/kernels/fmha_v2/setup.py

```diff
@@ -3055,8 +3055,9 @@ def get_cubin_header(kernel_traits, specs_names):
     cubins_dict = {}
     cubin_lens_dict = {}
     for kspec, fname, lname, kname in specs_names:
-        # only generate hopper cubin header
-        if generate_cu_trtllm and not 'sm90' in kname:
+        # only generate hopper head_size 128 cubin header
+        if generate_cu_trtllm and not ('sm90' in kname
```

Comment:
let us create a helper function so that  it can be reused in the following changes.

Reply:
And add some comments to describe why it is needed. 
Also make sure that the cubins can also be generated publicly or we need to find a way to always use the source codes otherwise the users might not be able to update the kernels as they want.